pen/views.py

```diff
@@ -0,0 +1,75 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.shortcuts import render, redirect
+from .models import Memo
+from django.contrib.auth import authenticate, logout
+from django.contrib.auth import login
+from django.views.decorators.http import require_http_methods
+
+
+def memo(request):
+    return render(
+        request,
+        'pen/memo.html'
+    )
+
+
+@require_http_methods(["POST"])
+def add_new_memo(request):
+    memo = Memo()
+    memo.body = request.POST["memo_body"]
+    memo.save()
+    return redirect('pen:memo')
+
+
+def show_all_memos(request):
+    memo = Memo.objects.all()
+    return render(
+        request,
+        'pen/show_all_memos.html',
+        context={
+            'memo': memo
+        }
+    )
+
+
+def login_view(request):
+    if request.method == "POST":
+        data = request.POST
+        user = authenticate(
+            username=data.get('user'),
+            password=data.get('password')
+        )
+
+        if user:
+            login(request, user=user)
+        return redirect('pen:home')
+    return render(
+        request,
+        "pen/login.html"
+    )
+
+
+def home(request):
```

Comment:
Ten widok chyba do niczego nie potrzebny. W tej chwili użytkownik po zalogowaniu jest przekierowywany na tę stronę, Jedyną korzyść jaką mamy z tego widoku to, że udostępnia przycisk wyloguj (ale może ten przycisk wrzucić na stronę z planszą - tylko że przycisk wyloguj nie powinien być już skanowalny, tylko zwykły - klikalny). Najlepiej chyba usunąć ten widok (razem z url-em i szablonem pod niego podpiętym), a użytkownika po zalogowaniu od razu przekierowywać na planszę memo.

Reply:
Przycisk wyloguj, wrzucony na stronę z planszą - zwykły - klikalny. Usunięty widok (razem z url-em i szablonem pod niego podpiętym), a użytkownik po zalogowaniu od razu przekierowany jest na planszę memo.